Core/Kernel/elxElastixMain.cxx

```diff
@@ -161,8 +161,8 @@ ElastixMain::ElastixMain()
  * ****************** Initialization of static members *********
  */
 
-ElastixMain::ComponentDatabasePointer ElastixMain::s_CDB             = 0;
-ElastixMain::ComponentLoaderPointer   ElastixMain::s_ComponentLoader = 0;
```

Comment:
`= 0` is not necessary here, because the type (`itk::SmartPointer`) has a default-constructor, which initializes the internal pointer.